docs/source/UIDocumentation.md

```diff
@@ -1,160 +1,238 @@
-Frontend Design 
-=======
-### Architecture
-```app.js``` hosts the main app and calls ```content.js```, which loads up the main app. This file contains a few functions:
-### ```content.js```
-(The top most bullet contains the name of the method and the inner bullets are params.)
-* ```openModal```
-* ```closeModal```
-* ```addNewLayer``` - invoked by ```handleClick``` and passed in a JS object with layer information. 
-  * layer
+Frontend Design
+========
+## Architecture
+(All javascript files described here can be found in the /ide/static/js/ directory of Fabrik)
+
+#### ```addCommentModal.js```
+(The bullets contain methods specific to the file along with their explanations.)
+
+```addCommentModal.js``` is responsible for adding comments into the interface. It has three methods:
+* ```handleClick```
```

Comment:
No explanation for this method? Having no explanation looks a bit weird while all other methods have them. Either move this method to the end of the list, or provide a brief explanation.